docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/clustering-for-high-availability.md

```diff
@@ -1,25 +1,31 @@
 # Clustering for High Availability
 
-Liferay DXP can scale to serve everything from the smallest to the largest web sites. Out-of-the-box, it's configured optimally for a single server environment. If one server isn't sufficient to serve your site's high traffic needs, DXP scales to the size you need.
+Liferay DXP can scale to serve everything from the smallest to the largest web sites. Out-of-the-box, it's configured optimally for a single server environment. If one server isn't sufficient to serve your site's high traffic needs, DXP scales to the size you need. <!-- I'd consider re-writing the last sentence to be a bit more inclusive of the different scenarios where clustering has a use case: "If you expect to have high traffic or want to improve the overall fault tolerance of your site, you should consider clustering." -->
 
-![Liferay DXP is designed to scale to as large an installation as you need.](./clustering-for-high-availability/images/01.png)
+![Liferay DXP is designed to scale to as large an installation as you need.](./clustering-for-high-availability/images/01.png) <!-- I know this image is an oldie but I find the diagram presented in the "example creating a dxp cluster" article a lot clearer. -->
```

Comment:
@jrhoun This image conveys how you can scale DXP in different ways (db cluster, lb cluster, es cluster), and it shows multitenancy as an added bonus :-)

Let's keep it. I love this image!

Reply:
Ha. Well. I didn't remove it or comment it out so we can keep it for now, but I think that expressing the multi-tenancy aspect dilutes the main thrust of the article - which is to convey a general clustering architecture overview. The "Unbreakable Liferay" and "...To infinity"  header also comes off a little casual to me - or maybe excessively optimistic? An alternative type of diagram that I'd prefer:

![image](https://user-images.githubusercontent.com/597214/80840159-89777100-8bb1-11ea-9c95-9e563979e213.png)

I don't think a clustering diagram would be exactly like that since you'd need to convey multiple nodes communicating with one another but I much prefer the straightforward style there.